SQL/Cleanup_patches/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches written as a post-release clean-up. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
+
+These patches should be run at the discretion of the projects with the full knowledge of the possible side effects of these.
```

Comment:
What would happen if a project decide not to run those cleanup patches?

Reply:
theoretically everything should still work fine in their codebase/database, these should only be clean-up

Reply:
added a line to explain